### ISSUE TITLE
Demand for API different from 'broote'

### ISSUE THREAD

**Issue by sekgobela-kevin** [sekgobela-kevin/webrute#6]
Currently 'webrute' seems to have API similar to that of 'broote'.
This could cause name collusions especially as webrute will be used along with 'broote'.
Its better to have different API but runner objects be used for compatibility with 'broote'.

It just takes wrapping the runner instances or classes into fuctions rather than creating runner instances directly.
Without doing so, it would be harder to differentiate 'webrute' code from that of 'broote'.

**Comment by sekgobela-kevin** [comment]
This needs to be done before README.md file can be updated.

**Comment by sekgobela-kevin** [comment]
THis taken from README.md
>
>### Progress and Issues
>- 'webrute' is currently unstable and not guaranteed to work.
>- Not having automated tests means there are lot of bugs.
>- Undefined goals and requirements led to poor implementation.
>- 'webrute' code can be easily be replaced with 'broote' with ease.

**Comment by sekgobela-kevin** [comment]
Here is how the API can be using functions wrapping runners.
```python

#def create_runner(*args, *kargs): ...

def create_async_runner(*args, *kargs): ...

def create_thread_runner(*args, *kargs): ...

def create_basic_runner(*args, *kargs): ...
```

**Comment by sekgobela-kevin** [comment]
`__init__.py` will need to be changed to not import the internal runner classes.

**Comment by sekgobela-kevin** [comment]
This issue is already is fixed([0ca20ee](https://github.com/sekgobela-kevin/webrute/commit/0ca20eebc9c08619a1345d56608c1ffd973b8d9f)) and can be closed.
